official/resnet/resnet_run_loop.py

```diff
@@ -344,10 +345,14 @@ def resnet_main(flags, model_function, input_function):
           'multi_gpu': flags.multi_gpu,
           'version': flags.version,
       })
+  benchmark_logger = (logger.BenchmarkLogger(flags.benchmark_log_dir)
+                      if flags.benchmark_log_dir is not None else None)
 
   for _ in range(flags.train_epochs // flags.epochs_between_evals):
     train_hooks = hooks_helper.get_train_hooks(
-        flags.hooks, batch_size=flags.batch_size)
+        flags.hooks,
+        batch_size=flags.batch_size,
+        benchmark_log_dir=flags.benchmark_log_dir)
```

Comment:
Is this the same logger as above? Or passing in the log dir will create a separate logging instance?

Reply:
It will create a separate logger instance. Since the logger instance does not keep any internal state, It is fine to create many instances. Alternatively, I could check the benchmark logger and get _logger instance from it, which is not ideal since it is trying to access an private field.

Reply:
SGTM.